src/LogExpert/Config/ConfigManager.cs

```diff
@@ -120,8 +120,8 @@ private Settings Load ()
         _logger.Info("Loading settings");
 
         string dir;
-
-        if (File.Exists(PortableModeDir + Path.DirectorySeparatorChar + PortableModeSettingsFileName) == false)
+        
+        if (!File.Exists(Path.Combine(PortableModeDir, PortableModeSettingsFileName)))

```

Comment:
Path.Combine, sometimes does not work with linux and can have unforeseen consequences => https://learn.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=net-9.0#remarks